.buildkite/ftr_oblt_serverless_configs.yml

```
@@ -6,6 +6,9 @@ disabled:
   - x-pack/test_serverless/functional/test_suites/observability/cypress/config_headless.ts
   - x-pack/test_serverless/functional/test_suites/observability/cypress/config_runner.ts
 
+  #Observability AI Assistant bulk import config, only run locally
```

Let's make this generic so we can use if for other types of tests if we need
```suggestion
  #Observability AI Assistant local-only tests
```